commands/commands.go

```diff
@@ -19,6 +23,106 @@ type Cmd struct {
 	Template string
 }
 
+type TomlConfig struct {
+	Env   string
+	Dev   ConsulFromFile
+	Prod  ConsulFromFile
+	Stage ConsulFromFile
+	QA    ConsulFromFile
+	West  ConsulFromFile
+	East  ConsulFromFile
+}
+
+type ConsulFromFile struct {
+	Consul    string
+	Ssl       bool   `toml:"ssl"`
+	SslCaCert string `toml:"ssl-ca-cert"`
+	SslCert   string `toml:"ssl-cert"`
+	SslKey    string `toml:"ssl-key"`
+	SslVerify bool   `toml:"ssl-verify"`
+	Token     string
+}
+
+func CheckConsulFile(inputFile string) string {
+	tempConsulFile := inputFile
+	if inputFile[:2] == "~/" {
+		user, _ := user.Current()
+		homeDir := fmt.Sprint(user.HomeDir + "/")
+		tempConsulFile = strings.Replace(inputFile, "~/", homeDir, 1)
+	}
+	_, err := os.Stat(tempConsulFile)
+	if err == nil {
+		return tempConsulFile
+	}
+	return ""
+}
+
```

Comment:
Might want to look at https://github.com/mitchellh/go-homedir for homedir detection. Biggest benefit is that it works in windows as well.